copy_req_files_to_vm.sh

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env bash
+
+# Script name: copy_proj_to_vm.sh
+# Description: Run this script BEFORE your run: setup_mysql.sh & setup_machine.sh
+# This script transfers those 2 setup files & the project files &
+# folders for the create-jazz-lyric Python Flask web app to the deployment machine.
+# Author: Kim Lew
+
+# --- Using Vagrant Configuration File ----------------------------------------
+# Assumes: vagrant-ssh-config was created as early step in README.
+
+VAG_SSH_CFG='vagrant-ssh-config'
+SRC_REQ_FILES_DIR='/Users/kimlew/code/vagrant_vm_centos7_create_jazz/'  # On Mac.
+SRC_PROJ_DIR='/Users/kimlew/code/vagrant_vm_centos7_create_jazz/pythonapp-createjazzlyric/'  # On Mac.
+DEST_REQ_FILES_AT_ROOT_DIR='/home/vagrant/'  # VM root directory.
+DEST_PROJ_DIR='/home/vagrant/pythonapp-createjazzlyric/'  # On VM.
+
+# --- Files needed at root directory on VM to set up VM. ---
+scp -F "${VAG_SSH_CFG}" "${SRC_REQ_FILES_DIR}"setup_mysql.sh vagrant@default:"${DEST_ROOT_DIR}"setup_mysql.sh
+scp -F "${VAG_SSH_CFG}" "${SRC_REQ_FILES_DIR}"setup_machine.sh vagrant@default:"${DEST_ROOT_DIR}"setup_machine.sh
+
+# --- Project files & folders for app, pythonapp-createjazzlyric ---
+# Note: scp -r  recursively copies entire directory, pythonapp-createjazzlyric
+# which includes:
+# README.md
+# conn_vars_dict.py
+# create_mysql_db.sql
+# Pipfile
+# Pipfile.lock
+# create_jazz_lyric.py
+# static
+# templates
```

Comment:
I wouldn't bother listing all these files in the comment, because as the contents of `pythonapp-createjazzlyric` change you'll have to remember to update this comment. I think it's sufficient to just say that you're copying the directory and all its contents.

Reply:
Good point about later maintenance of comment.